spec/laa_crime_schemas/structs/crime_application_spec.rb

```diff
@@ -22,6 +22,10 @@
           LaaCrimeSchemas::Validator.new(subject.to_json)
```

Comment:
Not sure it would go in this spec but I think it would be good to add a test so that breaks if the OFFENCE_CLASS_RANKING changes as a failsafe :)